middleware/functions/feedback-notion.ts

```diff
@@ -0,0 +1,109 @@
+import { Client } from "npm:@notionhq/client";
+import type { FeedbackSubmission } from "../../types.ts";
+
+// Collecting entries to this internal Notion page:
+// https://www.notion.so/denolandinc/192771466362800da55dc4552ad9a70c?v=192771466362803793d5000c0f522280
```

Comment:
do we want to publicise this?

Reply:
It's private so I thought there was no harm and it was convenient... but I'm happy to remove it if that's our preference

Reply:
I think probably better not shared just to discourage eyes from trying to look at it.

Reply:
I've removed it